app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -58,12 +61,13 @@
           self.props.onUserDropdownSelect(ui.item.value);
         }
       });
+
+      $('body').bind('click.closeMenu', this.closeMenu);
```

Comment:
Yeah, it basically listens for the  click event outside of the component itself to close.  

I'll implement `('body').bind('click.closeProvidedByEducatorDropdownMenu', this.closeMenu);`